custom_components/kohler/__init__.py

```diff
@@ -71,9 +117,6 @@ def setup(hass, config):
         )
         return False
 
-    for component in ["binary_sensor", "water_heater"]:
```

Comment:
I don't think I removed anything light related, only binary_sensor and water_heater were here before